data/blog/renewing-our-commitment-to-bitcoin.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: "Renewing Our Commitment to Bitcoin"
+date: '2024-12-05'
+tags: ['OpenSats', 'grants', 'bitcoin', 'wave']
+draft: false
+authors: ['arvin', 'dergigi', 'btcschellingpt']
+images: ['/static/images/blog/64-renewing-our-commitment-to-bitcoin.jpg']
+summary: "Renewing our support for projects advancing Bitcoin's future."
+---
+
+Over the past two years, we have made [41 announcements](/tags/bitcoin) from our
```

Comment:
I recommend referencing the # of grant recipients vs the # of announcements. I will start counting now.

Reply:
Good suggestion, but I think it is okay to leave as is for this particular post. As you count, you may find that a particular grantee received a project grant early on, then later may have received an LTS grant; in cases like this you will have two grant instances within two announcements, but one grantee. For simplicity, let's leave it as is.

Reply:
@Arvin21M Oops. Just saw your comment suggesting that I do not count. 

If you are still interested in making the change, you could say "Over the past two years, we have issued over 100 grants including 31 long-term grants to support individual contributors and projects working to grow the Bitcoin ecosystem."

You could also say "Over the past two years, we have issued 109 grants including 31 long-term grants to support individual contributors and projects working to grow the Bitcoin ecosystem."